Main.cs

```diff
@@ -96,14 +98,7 @@ public List<Result> Query(Query query)
                 {
                     // Add score condition as there could be multiple windows with the same title, eg. same web pages opened
                     if (cache.Value.Title == results[i].Title && results[i].Score == cachedWindowsScore)

```

Comment:
Sorry I am not sure about results[i].Score will equal to cachedWindowsScore.
If it is cachedWindows, shoudn't it return directly?

Reply:
I think this part of code is for boosting score when query is not cached, but cached windows are searched by them, so make their result higher (Actually I don't think it is a great idea unless the query search is empty, because people are supposed to access the cached windows by the cached action keyword).

Reply:
Oh yeah you are right lol. I am just trying to figure out a way to make cached window on top against frequently used ones. Hmmm

Reply:
As well as removing duplicates if just comparing on title

Reply:
yeah agree with you. I think we just need to change the score when the query is empty (actually I mean using the const variable instead of direct score😂), but I don't think we should change the score when query is not empty, because cachedwindow are supposed to be accessed by customized keyword.

Reply:
I can do that for you, and you can take a look.
Thank you for the idea!